assets/queries/cloudFormation/filter_and_alarm_exist_for_cloudtrail_config_changes/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_CloudTrail_Configuration_Changes",
+  "queryName": "Filter And Alarm Not Exists For CloudTrail Configuration Changes",
+  "severity": "LOW",
+  "category": "Monitoring",
+  "descriptionText": "A log metric filter and alarm should exist for CloudTrail Configuration Changes",
+  "descriptionUrl": "https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudwatch-alarms-for-cloudtrail.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline